src/app/GlobalStyles.tsx

```diff
@@ -7,6 +7,11 @@ export default function GlobalStyles() {
   return (
     <Global
       styles={css`
+        html {
+          scroll-behavior: smooth;
+          scroll-padding-top: 79px;
```

Comment:
where is 79px from?

Reply:
It is the addition of the height of `NavBar` (55px) and an arbitrary gap (24px).


```
// /src/components/NavBar.tsx

export const Nav = styled.nav`
  display: flex;
  justify-content: space-between;
  padding: 12px 96px 16px 96px;
  align-items: end;
  border-bottom: 1px solid ${Color.gray300};
`

...

return (
    <Nav>
      <Logo href="/">
        <Image src="/images/logo.png" width={100} height={26} alt="KIXLAB Logo" />
        <span>KAIST Interaction Lab</span>
      </Logo>
...

```


![image](https://github.com/kixlab/website/assets/100757008/a26073cd-a618-4282-a7ee-2085276d4963)

I'm unsure if this is the best practice or not .. 😂

Reply:
Then I would define a const for the `NavBar` height and share them across different css.
```tsx
const NAV_BAR_HEIGHT = 55 (or 56)

export const Nav = styled.nav`
  display: flex;
  padding: 0 96px;
  height: ${Nav}px;
  justify-items: center;
  ...
`

const GlobalStyle = css`
   html {
       scroll-behavior: smooth;
       scroll-padding-top: ${NAV_BAR_HEIGTH + 24}px;
  ...
  }
`
```

Plus, I heard that even heights and widths are often preferred.

Reply:
Thanks for the feedback! I defined a const `NAV_BAR_HEIGHT` as 56 in the `NavBar.tsx` and export it to use in `GlobalStyles.tsx` as well.